nilearn/decoding/decoder.py

```diff
@@ -452,11 +443,12 @@ def fit(self, X, y, groups=None):
 
         parallel = Parallel(n_jobs=self.n_jobs, verbose=2 * self.verbose)
 
-        for i, (class_index, coef, intercept, y_info, params, scores) \
-                in enumerate(parallel(delayed(self._cache(_parallel_fit))(
-                    self.estimator, X, y[:, c], train, test,
-                    self.param_grid, self.is_classif, scorer,
-                    self.mask_img_, c, self.screening_percentile_)
+        for i, (class_index, coef, intercept,
+                y_info, params, scores) in enumerate(
+                    parallel(delayed(self._cache(_parallel_fit))(
```

Comment:
Can the argument for `parallel()` or a subset of it, be assigned to a well-named variable? That would improve comprehension & readability.